clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -1348,7 +1348,6 @@ public void testOnJoinPrepareWithOffsetCommitShouldKeepJoinAfterRebalanceTimeout
             String memberId = "consumer-42";
 
             Timer pollTimer = time.timer(100L);
```

Comment:
i think we also need to change time to `Timer pollTimer = time.timer(0L);` if we are going to remove `time.sleep(150);` below.  
i think the test hang here.